core/capabilities_test.go

```diff
@@ -339,12 +339,40 @@ func TestCapability_CompatibleWithNetCap(t *testing.T) {
 	orch.version = "0.4.0"
 	assert.False(bcast.CompatibleWith(orch.ToNetCapabilities()))
 
-	// broadcaster is not compatible with orchestrator - the same version
+	// broadcaster is compatible with orchestrator - the same version
 	orch = NewCapabilities(nil, nil)
 	bcast = NewCapabilities(nil, nil)
 	bcast.constraints.minVersion = "0.4.1"
 	orch.version = "0.4.1"
 	assert.True(bcast.CompatibleWith(orch.ToNetCapabilities()))
+
+	// broadcaster is compatible with orchestrator - no pre-release min version suffix
+	orch = NewCapabilities(nil, nil)
+	bcast = NewCapabilities(nil, nil)
+	bcast.constraints.minVersion = "0.4.1"
+	orch.version = "0.4.1-0.21000000000000-06f1f383fb18"
```

Comment:
Oh if our versions are like this and not like `ai-video` or `some-specific-feature` then the semver comparison might be great